src/main/java/tutoraid/model/util/SampleDataUtil.java

```diff
@@ -24,34 +24,33 @@
  */
 public class SampleDataUtil {
 
-    public static final Progress DEFAULT_PROGRESS = new Progress("No Progress");
     public static final PaymentStatus DEFAULT_PAYMENT_STATUS = new PaymentStatus(false);
 
     public static Student[] getSampleStudents() {
         return new Student[] {
             // All details available
             new Student(new StudentName("Alex Yeoh"), new Phone("87438807"),
                     new ParentName("Mr Yeoh"), new Phone("93726483"),
-                    DEFAULT_PROGRESS, DEFAULT_PAYMENT_STATUS),
+                    new ProgressList(), DEFAULT_PAYMENT_STATUS),
```

Comment:
Would it be better to declare `new ProgressList()` as a final variable at the start and just include this final variable in the following statements instead?

Reply:
Right I agree, can leave it as it is then!